File: src/main/java/com/aliiensmp/aliienResize/Config/Records/GuiData.java
package com.aliiensmp.aliienResize.Config.Records;

import org.bukkit.inventory.ItemFlag;
import java.util.List;

public record GuiData(
        String material,
        int slot,
        int page,
        String name,
        List<String> lore,
        List<String> loreWithoutPerm,
        int modelData,
        List<ItemFlag> itemFlags
) {}

File: src/main/kotlin/com/aliiensmp/aliienResize/config/data/GuiData.kt
package com.aliiensmp.aliienResize.config.data

import org.bukkit.inventory.ItemFlag

data class GuiData(
    val material: String,
    val slot: Int,
    val page: Int,
    val name: String,
    val lore: List<String>,
    val loreWithoutPerm: List<String>,
    val modelData: Int,
    val itemFlags: List<ItemFlag>
)